src/widgetastic_patternfly/__init__.py

```diff
@@ -513,7 +513,7 @@ class BootstrapNav(Widget):
     TEXT_MATCHING = './/li/a[text()={txt}]'
     PARTIAL_TEXT = './/li/a[contains(normalize-space(.), {txt})]'
     ATTR_MATCHING = './/li/a[@{attr}={txt}]'
-    TEXT_DISABLED = './/li[contains(@class, "disabled")]/a[text()={txt}]'
+    TEXT_DISABLED = './/li[contains(@class, "disabled")]/a[contains(text(), {txt})]'
```

Comment:
There's an assumption baked into making this a contains match, that I don't think we want.

Note the behavior of the `select` and `is_disabled` methods, and how they inspect the `text` argument to determine if its a partial_match object. If it is, then they use the `PARTIAL_TEXT` locator, otherwise the `TEXT_MATCHING` locator.

I think you need to apply the same behavior here, and define a separate `TEXT_PARTIAL_DISABLED` locator, used when the passed text argument is partial match.